templates/object-detection/template.yaml

```diff
@@ -40,6 +40,7 @@ spec:
           title: DETR Model Server
           description: The model server container to use. The deployed model on the server must support object detection.
           type: string
+          default: detr-resnet-50
```

Comment:
the default one we are using is detr-resnet-101

Reply:
Fixed